### PR TITLE
Connect front-end to sorted-by-provider and date index

### PR DESCRIPTION
**Pull request by FKolwa**
This PR enables the front-end to properly display the index files for sort-by-provider-and-date.

### CODE REVIEW

**Review by miyunari — APPROVED** [review]
Nice! Can't wait to see it in action! :smiley:

File: src/app/Pages/Browser/AWS.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ const AWS: React.FunctionComponent<{title: string}> = ({title}) => {
           <Title headingLevel="h1"  size={TitleSizes['4xl']}>AWS Image Browser</Title>
         </Bullseye>
       </PageSection>
-      <ImageDataTable tableColumns={columns} />
+      <ImageDataTable tableColumns={columns} pathPrefix={'https://poc.imagedirectory.cloud/images/v1/idx/list/sort-by-date-aws'}/>
```
Comment (miyunari) [inline src/app/Pages/Browser/AWS.tsx:36]: Ah - do we need to remove the `poc` sub-domain at all places? :thinking:

Comment (FKolwa) [reply src/app/Pages/Browser/AWS.tsx:36]: Jap! ONce the infrastructure changes are in place (redirect without the poc subdomain) we can remove all of the tags ;)

Comment (FKolwa) [reply src/app/Pages/Browser/AWS.tsx:36]: I hope it is this simple: https://github.com/redhatcloudx/infrastructure/pull/29

**Pull request merged**